tests/DCPS/QoS_XML/dump/qos_dump.cpp

```diff
@@ -101,6 +101,13 @@ int ACE_TMAIN (int, ACE_TCHAR *[])
               ++retval;
             }
 
+          if (tp_qos.durability_service.history_kind != DDS::KEEP_LAST_HISTORY_QOS)
+          {
+            ACE_ERROR ((LM_ERROR, "PARSEXML - "
+                  "get_topic_qos returned an invalid type durability_service history_kind.\n"));
+            ++retval;
+          }
```

Comment:
The formatting of this test doesn't follow OpenDDS style guide. We may want to update it some other time.